ios/app/app/API/EndPoint.swift

```diff
@@ -8,18 +8,23 @@
 import Foundation
 import Combine
 
+enum Commands: String {
+    case transactions = "/transactions"
+    case users = "/users"
+}
+
 struct Endpoint<DataType: Codable> {
-    var path: String
+    var path: Commands
```

Comment:
maybe create additional abstract level will be better choice.
something like
```
protocol EndpointProtocol {
    associatedtype DataType: Codable
    var path: String { get }
    var queryItems: [URLQueryItem] { get set }
}
```

and in this case, you can own any Endpoint struct with self contained attributes as you wish. for example.
```
struct GetUser<DataType: Codable> : EndpointProtocol {
   var path: String = "users" // or Commands that you created.
   var queryItems = [URLQueryItem]()
}
```

any thoughts?

Reply:
I am just concerned we might be abstracting too much for a small project.

Reply:
yo, don't worry, make it scalable, you never know where this project could ends. @psymoon right? I don't know if simon will get addicted to add more api and stuffs. you know. lol

Reply:
Yeah, we should go for scalability and flexibitliy :).